runtime/sema/publicaccount_type.go

```diff
@@ -116,9 +116,9 @@ var PublicAccountType = func() *CompositeType {
 var PublicAccountKeysType = func() *CompositeType {
 
 	accountKeys := &CompositeType{
-		Identifier: AccountKeysTypeName,
-		Kind:       common.CompositeKindStructure,
-		importable: false,
```

Comment:
I renamed `importable` to `ImportableWithoutLocation` to make it accessible and better describe the effect this value has on behavior.

This may be unnecessary: if CompositeTypes with `false` for `importable` are never transmitted then there's no need to encode them.